ui/src/Context/NotificationContext.js

```diff
@@ -0,0 +1,38 @@
+import { createContext, useContext, useState } from "react";
+import { NOTIFICATION_TIMEOUT } from "../Constant/constant";
+
+export const NotificationContext = createContext({
+  createNotifcation() {},
+  notificationQueue: [],
+});
+
+export const useNotificationContext = () => useContext(NotificationContext);
+
+const NotificationProvider = ({ children }) => {
+  const [notificationQueue, setNotificationQueue] = useState([]);
+
+  /**
+   *
+   * @param {type : string, message : string} notificationContent
+   */
+  const createNotifcation = (notificationContent) => {
+    setNotificationQueue((prev) => [...prev, notificationContent]);
+
+    setTimeout(() => {
+      setNotificationQueue((prev) => {
+        let [_, ...data] = prev;
```

Comment:
we don't need `_ `here. You can just keep it: `let [, ...data] = prev` and it will do the same job.